modules/11_aerosols/exoGAINS2025/input/files

```diff
@@ -0,0 +1,10 @@
+emi2020_sectGAINS_sourceCEDS.cs4r
+emifacs_sectGAINS_sourceCEDS.cs4r
+f11_emifacs_sectREMIND_sourceCEDS.cs4r
+emi2020_sectGAINS_sourceGAINS.cs4r
+emifacs_sectGAINS_sourceGAINS.cs4r
```

Comment:
are all emi* files generated by input data process? If so, you do not need to add those to cfg$files2export$start, Distribution would happen automatically